src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const prise = 40;
+
+  if (days < shortTerm) {
+    return days * prise;
+  } else if (days < longTerm) {
+    return days * prise - shortTermDiscount;
```

Comment:
The variable `prise` is used here, but it should be corrected to `price` to ensure the calculation is accurate.